controllers/projectController.go

```diff
@@ -0,0 +1,108 @@
+package controllers
+
+import (
+	. "clokify/models"
+	. "clokify/services"
+	"encoding/json"
+	"io"
+	"log"
+	"strconv"
+
+	. "clokify/types"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
```

Comment:
you are using gin package. it is good. but, it is not our project scope. let's go with it.